mlops/kubeflow/README.md

```diff
@@ -0,0 +1,66 @@
+# Kubeflow Pipeline Components for ART
+
+Kubeflow pipeline components are implementations of Kubeflow pipeline tasks. Each task takes
+one or more [artifacts](https://www.kubeflow.org/docs/pipelines/overview/concepts/output-artifact/)
+as input and may produce one or more
+[artifacts](https://www.kubeflow.org/docs/pipelines/overview/concepts/output-artifact/) as output.
+
+
+**Example: ART Components**
+* [Adversarial Robustness Evaluation - FGSM - PyTorch](robustness_evaluation_fgsm_pytorch)
+
+Each task usually includes two parts:
+
+Each component has a component.yaml which will describe the functionality exposed by it, for e.g.
+
+```
+name: 'Adversarial Robustness Evaluation'
+description: |
+  Perform an evaluation of the adversarial robustness using the Fast Gradient Sign Method of the Adversarial Robustness Toolbox (ART).
```

Comment:
@Tomcli Please see my comment below on the disclaimer that I think we should add - it should go here, too.